server/src/internalClusterTest/java/org/elasticsearch/action/admin/indices/shards/IndicesShardStoreRequestIT.java

```diff
@@ -48,9 +49,13 @@ protected Collection<Class<? extends Plugin>> nodePlugins() {
         return Arrays.asList(MockFSIndexStore.TestPlugin.class);
     }
 
+    private static IndicesShardStoresResponse execute(IndicesShardStoresRequest request) {
+        return client().execute(TransportIndicesShardStoresAction.TYPE, request).actionGet(10, TimeUnit.SECONDS);
```

Comment:
Why use `IndicesAdminClient#shardStores(IndicesShardStoresRequest)`? It seem to be easier to read than a generic `client.execute(..)`?